### DISCUSSION TITLE
Can anyone make these extension ideas?

### DISCUSSION THREAD

**Discussion by ghost** [TurboWarp/extensions#1264] [Ideas]
https://scratch.mit.edu/projects/888620595/ a list of extension ideas not made by me, but I NEED THEM!!!(after someone makes extensions below:
Also can anyone make a cloud list extension(Need this most)? Or a recording extension(Need this secind most)? I also need a cloud variable system that does not need the project to be packaged(tried using websocket, not that good) where you can access cloud variables with a certain project id(if cloud list not possible).

Edit: THE REASON WHY I DONT CODE A CLOUD LIST IS BECAUSE YOU CANNOT ACCESS TURBOWARP CLOUD IN EDITOR(unless u use websocket which is very buggy(I tried)) SO PLS AAT LEAST MAKE A CLOUD VARIABLE EXTENSION WITH ANOTHER SERVER.

**Comment by PURPL3-CAT** [comment]
It may be worth looking at some existing extensions. Many of them do similar things to what you listed. Nevertheless, those are good ideas.

**Comment by MA-master-sec** [comment]
![image](https://github.com/TurboWarp/extensions/assets/126303227/fbabc66c-10d4-46cb-b5c7-4ade6a3d0e71)

**Reply by MA-master-sec** [reply]
you could try`try`&`catch`

**Comment by HirujaEdurapola** [comment]
Creating a cloud list extension is not possible (as i know) cuz it takes lots of storage in the turbowarp's servers. This can be avoided by setting limits like maximum list length is 100 or something. But even though you create something with all the limits it will still be useless cuz cloud variables/list can only handle numbers.

However you can consider the following code that can be used to create cloud lists
[Cloud List.sprite3](https://github.com/TurboWarp/extensions/files/14317731/Cloud.List.sprite3.zip)
 
Make sure to add the following .txt to the List called LetterList inside the sprite.
[LetterList.txt](https://github.com/TurboWarp/extensions/files/14317739/LetterList.txt)

Hope That Helps. Thanks!

**Reply by ghost** [reply]
wont work, because turbowarp cant access cloud servers in turbowarp editor, so a separate extension compatible in turbowapr would be great.

**Reply by Goshanchick** [reply]
You can use the POST and GET commands, as well as send signals to the server for saving in the cloud. For a one-time multi-user session (for example, CS), you can use a WebSocket such as CloudLink and others.

**Reply by ghost** [reply]
well idk how to use websocket to read the cloud variable's value. Also what do you send to edit and receive a cloud variables value in turbowarp cloud servers?

**Comment by SharkPool-SP** [comment]
I made a recording extension. It is on my gallery 

https://sharkpool-sp.github.io/SharkPools-Extensions/
